tests/api/fixtures/aws.py

```diff
@@ -128,6 +128,29 @@ def airflow_prod_policy(iam):
     return result["Policy"]
 
 
+@pytest.fixture(autouse=True)
+def bedrock_policy(iam):
+    result = iam.meta.client.create_policy(
+        PolicyName="bedrock-policy-name",
```

Comment:
Needs changing when policy name is agreed

Reply:
Will be `analytical-platform-bedrock-integration`